### DISCUSSION TITLE
Dealing with files

### DISCUSSION THREAD

**Discussion by idc77** [ent/ent#2429] [Q&A] [answered]
I believe in v0.9 there was a piece of documentation about dealing with uploaded files automatically.
I tried searching for all kinds of terms (file, upload, storage) but it seems that it's just not present in v0.10's docs anymore.

It was something about storing a file in some kind of 3rd party service but all this being represented in a field definition or similar.

**Comment by crossworth** [comment, marked as answer]
Maybe https://entgo.io/blog/2021/11/1/sync-to-external-data-systems-using-hooks

**Reply by idc77** [reply]
that was it thanks
